generator/internal/evacuation_lrp_processor.go

```diff
@@ -151,8 +141,3 @@ func (p *evacuationLRPProcessor) evacuateClaimedLRPContainer(logger lager.Logger
 
 	p.containerDelegate.DeleteContainer(logger, lrpContainer.Container.Guid)
 }
-
-func writeToStream(streamer log_streamer.LogStreamer, msg string) {
-	fmt.Fprintf(streamer.Stdout(), msg)
-	streamer.Flush()
```

Comment:
We don't need a flush?

Reply:
No, we don't need to. We removed the usage of the log stream object entirely, it's entirely unnecessary, would also break log rate metrics, and shouldnt be log rate limited, so we're just directly using the metron agent object directly now.

Reply:
We replaced this in a lot of places in executor too. Everywhere but health checks and that doesn't have the same metrics problem cause it's the same object.